api/controllers/v1/slack.ts

```diff
@@ -144,7 +144,8 @@ export const getDeployableJobs = async (values, entitlement, branchRepository: B
     );
 
     if (!aliases || aliases.length === 0) {
-      if (non_versioned) {
+      // handle non-versioned repos AND repos where only 1 version is active where there are no aliases
+      if (non_versioned || (!publishOriginalBranchName && urlSlug === null)) {
```

Comment:
nit: this is the same logic (and comment) as line 167. perhaps toss it into a helper function?

Reply:
Reasonable nit! Broadly we should refactor all of this logic so it isn't hardcoded in the way it currently is, but that's probably out of the scope here. You'll note it's a SLIGHTLY different comment, since this mentions aliases while the other doesn't, but it is a very subtle difference :)

Reply:
Agreed on the refactor. I believe we're tracking some similar work in [DOP-3079](https://jira.mongodb.org/browse/DOP-3079), and Raymund and I recently had a [discussion on GitHub](https://github.com/mongodb/snooty/pull/644#discussion_r906333253) for a similar edge case bug fix in `snooty`